### ISSUE TITLE
Physical iPad Air 2 (Software Version 15.8) will not run new MAUI app

### ISSUE THREAD

**Issue by jeff-eats-pubsubs** [dotnet/maui#19764]
### Description

Hey all, 

Would like to give a big thank you to the MAUI team, I am enjoying MAUI through it all. I just began testing on on an iPad Air 2 (15.8) and was not getting anywhere with my current application, so I created a new MAUI application and cannot get that running. Seems worth reporting.

Let me know if there is anything else I can provide. Please excuse me if there is anything I neglected.

### Steps to Reproduce

Out of the box MAUI app. Using VS2022 Preview (17.9.0 preview 2.1) on Windows 10. The app builds and deploys to iPad Air 2 (15.8) never runs past the splash screen before (I think) watchdog ends it. 

1. Create a File > New .NET MAUI App
2. Connect Physical iPad Air 2 (15.8)
3. Run in debug or release (with or without debugging) and observe the bug 🐞

Expected outcome: It makes it through splash screen.
Actual outcome: It doesn't.

#### What I have tried ####
- Running with Hot Reload Disabled.
- Just deploying to the device.
- Using .NET 7.

Expected outcome: It makes it through splash screen.
Actual outcome: It doesn't.

### Version with bug

8.0.3

### Is this a regression from previous behavior?

Just tested in NET7 and appears to be the same. 

### Last version that worked well

Unknown/Other

### Affected platforms

iOS

### Affected platform versions

15.8

### Did you find any workaround?

Not yet

**Comment by drasticactions** [comment]
Based on your description, my guess would be that you're using "Hot Restart" and directly connecting your iPad to your Windows computer. Based on the iOS version, it could be too old to support that.

It would need to be addressed in Visual Studio. It can't be addressed in this repo.

**Comment by jeff-eats-pubsubs** [comment]
@drasticactions But also it doesn't work running without debugging or just deploying to device

**Comment by drasticactions** [comment]
“Hot Restart” is not “Hot Reload”

Hot Restart is what lets you deploy an iOS app from a Windows device without a connected Mac, which I assume is what you’re using. Are you or do you have a connected Mac?

**Comment by nbaulesglobalsolutions** [comment]
yes. just the ipad. no Mac

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by nbaulesglobalsolutions** [comment]
no connected Mac. only a connected HP which i've loaded my icloud and apple files onto

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by jeff-eats-pubsubs** [comment]
> “Hot Restart” is not “Hot Reload”
> 
> Hot Restart is what lets you deploy an iOS app from a Windows device without a connected Mac, which I assume is what you’re using. Are you or do you have a connected Mac?

Ah ok cool, thank you. You are absolutely correct just using Windows as of right now. So even just deploying to the device would be using Hot Restart?

**Comment by jeff-eats-pubsubs** [comment]
@drasticactions Thanks again

**Comment by jeff-eats-pubsubs** [comment]
Alright so...
* I got an M2 Mac Mini. 
* Installed XCode. 
   * Latest stable version of mono is unsigned so had to manually install mono. 
   * And for better or worse I went with the current preview because it was signed.
* Connected Mac to VS for Windows
   * I can get a new MAUI app running on simulators (iOS 15 and 17.2 on iPads and iPhone SE so far)

🙃 I still cannot get past the splash screen on my iPad Air 2
